ghga_connector/core/main.py

```diff
@@ -323,3 +320,26 @@ def download_parts(
             file.write(part)
             downloaded_size += len(part)
             queue.task_done()
+
+
+def get_wps_token(max_tries: int, message_display: AbstractMessageDisplay) -> List[str]:
+    """
+    Expect the work package id and access token as a comma separated string
```

Comment:
```suggestion
    Expect the work package id and access token as a colon separated string
```
That's where I got the comma from ...